try {
  throw new Error('catch-placeholder-error-param');
} catch (_) {
  console.log(_);
}
